src/pkg/packaging/dir.proj

```diff
@@ -160,18 +160,19 @@
 
   <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage">
     <ItemGroup>
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\project.json" />
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.Extensions.DependencyModel\project.json" />
+      <PackageProjects Condition="'$(OS)' == 'Windows'" Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\Microsoft.DotNet.PlatformAbstractions.csproj" />
```

Comment:
It is because these projects cannot be built outside of Windows -- they rely on the .NET Framework targeting pack. Previously, we were skipping the "net45" portion of the build, but still running `dotnet pack --no-build` on them in this phase. With the 2.0 SDK, the `dotnet pack` command fails because the .NET Framework targeting pack can't be found, even though we've passed `--no-build`.